books/free-programming-books-ta.md

```diff
@@ -41,6 +41,7 @@
 
 * [எளிய தமிழில் CSS](https://freetamilebooks.com/ebooks/learn-css-in-tamil/) - Kaniyam Foundation (PDF)
 * [எளிய தமிழில் HTML](https://freetamilebooks.com/htmlbooks/html-book/Learn-HTML-in-Tamil.html) = Th . Srinivasan (PDF)
+* [எளிய தமிழில் HTML](https://noolaham.net/project/51/5090/5090.pdf) - வே.நவமோகன் (PDF)
```

Comment:
this is from 2001 !

Reply:
@eshellman  I agree that the book is quite dated. However, it's important to note that it remains an outstanding resource for beginners learning HTML in Tamil Language.